setup.py

```diff
@@ -37,7 +37,7 @@
 if __name__ == '__main__':
     setup(
         name='topiary',
-        version="0.0.6",
+        version="0.0.8",
```

Comment:
I found the code changed locally from some forgotten branch, I guess that
was 0.0.7?

On Mon, Aug 24, 2015 at 3:39 PM, Tavi Nathanson <notifications@github.com>
wrote:

> In setup.py
> <https://github.com/hammerlab/topiary/pull/20#discussion_r37794456>:
>
> > @@ -37,7 +37,7 @@
> >  if __name__ == '__main__':
> >      setup(
> >          name='topiary',
> > -        version="0.0.6",
> > +        version="0.0.8",
>
> What happened to 0.0.7?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hammerlab/topiary/pull/20/files#r37794456>.
>